src/components/help/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  FaUser,
+  FaCogs,
+  FaWallet,
+  FaClipboardList,
+  FaBook,
+} from 'react-icons/fa';
+
+import styles from './styles.module.scss';
+
+export const Help = () => {
+  return (
+    <section className={styles.cards_wrapper}>
```

Comment:
I suggest using of destructing functionality of es6.

`const {className1, className2} = styles`

Reply:
Css modules are not in camelcase, so if I try to destructure this object and use default variables, our eslint settings says to use camelcase instead.